### ISSUE TITLE
500 HttpException Attempt to read property "photo_id" on null

### ISSUE THREAD

**Issue by 31ank** [LycheeOrg/Lychee-Docker#197]
Hi,
so I tried to setup lychee via Docker and I get the error ```500 HttpException Attempt to read property "photo_id" on null``` when trying to view an album.
In the logs the following error is documented:
```Illuminate\Foundation\Bootstrap\HandleExceptions::Illuminate\Foundation\Bootstrap\{closure}:215 Attempt to read property "photo_id" on null```

The lychee version i'm running: Lychee 5.2.1

My setup script:
```
docker run -d \
--name=lychee \
-v /home/user/lychee/conf:/conf \
-v /home/user/lychee/uploads:/uploads \
-v /home/user/lychee/sym:/sym \
-e PHP_TZ=America/New_York \
-e TIMEZONE=America/New_York \
-e APP_URL=http://localserveraddress:90 \
-p 90:80 \
lycheeorg/lychee
```
What I did: 
1. created an album
2. uploaded photos
3. error gets displayed

I use the local server address as APP_URL since I don't plan to reach the site from outside of my network.

**Comment by d7415** [comment]
This has just been fixed in Lychee. It will be in the `nightly` builds this evening, and in the next stable release.

**Comment by d7415** [comment]
Now fixed in `:latest` (v5.2.2)
